module-9/server/config.js

```diff
@@ -0,0 +1,9 @@
+// Config settings for the app
+import dotenv from 'dotenv';
+dotenv.config();
+
+export const PORT = process.env.PORT || 3001;
+export const MONGO_URI = process.env.MONGO_URI;
```

Comment:
Make sure that you're exporting the correct environment variable. It should be `MONGODB_URI` instead of `MONGO_URL` to avoid any connection issues.

```suggestion
export const MONGO_URI = process.env.MONGODB_URI;
```